embrace-android-features/src/main/kotlin/io/embrace/android/embracesdk/internal/injection/DataCaptureServiceModuleImpl.kt

```diff
@@ -83,6 +83,7 @@ internal class DataCaptureServiceModuleImpl @JvmOverloads constructor(
         if (configService.autoDataCaptureBehavior.isUiLoadPerfCaptureEnabled()) {
             createActivityLoadEventEmitter(
                 uiLoadEventListener = uiLoadTraceEmitter,
+                autoTraceEnabled = configService.autoDataCaptureBehavior.isUiLoadPerfAutoCaptureEnabled(),
```

Comment:
nit: `configService.autoDataCaptureBehavior.isUiLoadPerfCaptureEnabled()` could be extracted to a local variable